LayoutTests/platform/glib/TestExpectations

```diff
@@ -1083,6 +1083,8 @@ media/media-source/media-managedmse-memorypressure-inactive.html [ Timeout ]
 media/media-source/media-managedmse-eviction.html [ Timeout ]
 # No AirPlay on glib platforms.
 media/media-source/media-managedmse-airplay.html [ Skip ]
+# No MSE in a worker on glib platforms for now.
+media/media-source/media-managedmse-worker.html [ Skip ]
```

Comment:
Maybe we should have made all the the worker tests in a specific media-source/worker folder.

Reply:
just moved it to a new worker directory